### PR TITLE
Env util helper `get_path`

### PR DESCRIPTION
**Pull request by beshrkayali**
Based on a suggestion from @flaeppe, this adds a method to the environment parsing helper that allows parsing strings as a `pathlib.Path`.

### CODE REVIEW

File: src/bananas/environment.py [hunk 1]
```diff
@@ -264,6 +275,17 @@ def get_bool(self, key: str, default: None = None) -> Optional[bool]:
...
 
+    @overload
+    def get_path(self, key: str, default: Path) -> Path:
+        ...
+
+    @overload
+    def get_path(self, key: str, default: None = None) -> Optional[Path]:
+        ...
+
+    def get_path(self, key: str, default: object = None) -> object:
```
Comment (antonagestam) [inline src/bananas/environment.py:286]:
```suggestion
    def get_path(self, key: str, default: Optional[Path] = None) -> Optional[Path]:
```

Would be more correct, but I see this is following the other implementations here.

**Review by antonagestam — APPROVED** [review]
Nice! See related plans to improve the environment getters: https://github.com/5monkeys/django-bananas/issues/47

File: src/bananas/environment.py (hunk 1 above)
Comment (beshrkayali) [reply src/bananas/environment.py:286]: Ah I see, was a bit difficult cumbersome to figure out what's going on here but now I see it makes sense :slightly_smiling_face:

File: src/bananas/environment.py [hunk 2]
```diff
@@ -272,7 +272,7 @@ def get_bool(self, key: str, default: U) -> Union[bool, U]:
     def get_bool(self, key: str, default: None = None) -> Optional[bool]:
         ...
 
-    def get_bool(self, key: str, default: object = None) -> object:
```
Comment (antonagestam) [inline src/bananas/environment.py:264]: This was probably unintentional, right? :)

**Review by antonagestam — CHANGES_REQUESTED** [review]

File: src/bananas/environment.py (hunk 2 above)
Comment (beshrkayali) [reply src/bananas/environment.py:264]: Yep!

**Comment by antonagestam** [comment]
I just recalled that this is the only thing we have type tests for, which is nice to have because it verifies that the exposed typing is correct. Would you mind adding a case for Path as well? https://github.com/5monkeys/django-bananas/blob/master/tests/test_env.yaml

**Comment by antonagestam** [comment]
We want to test that:

- Non-`Path` default is a type error
- Omitting a default returns `Path | None`
- Passing a default returns `Path`

**Pull request closed without merging**